app/shop.py

```diff
@@ -0,0 +1,10 @@
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            products: list,

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]